scripts/proto-gen.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+#
+# Update the generated code for protocol buffers in the Tendermint repository.
+# This must be run from inside a Tendermint working directory.
+#
+set -euo pipefail
+
+# Work from the root of the repository.
+cd "$(git rev-parse --show-toplevel)"
```

Comment:
if we're not in a git repository this won't error because errexit doesn't propogate to subshells.

Reply:
Yeah, that's fine. The rest of the build will harmlessly fail if we're not.